widget/playground/src/containers/StyleLayout/StyleLayout.Preset.tsx

```diff
@@ -0,0 +1,160 @@
+import type { ColorsTypes, PresetTypes } from './StyleLayout.types';
+import type { WidgetColors } from '@rango-dev/widget-embedded';
+
+import { Button, Divider, Switch, Typography } from '@rango-dev/ui';
+import React, { useState } from 'react';
+
+import { CUSTOM_THEMES } from '../../constants';
+import { useConfigStore } from '../../store/config';
+import { shallowEqual } from '../../utils/common';
+
+import {
+  Collapse,
+  Color,
+  ColorsContent,
+  PresetContent,
+  PresetTheme,
+  Row,
+} from './StyleLayout.styles';
+
+const Colors = (props: ColorsTypes) => {
+  return (
+    <ColorsContent>
+      <Color style={{ backgroundColor: props.primary }} />
+      <Color
+        style={{
+          backgroundColor: props.secondary,
+          position: 'absolute',
+        }}
+      />
+      <Color
+        style={{
+          backgroundColor: props.background,
+          position: 'relative',
+        }}
+      />
+    </ColorsContent>
+  );
+};
+
+const FIX_HEIGHT = 41;
+const NUMBER_ITEMS_2_COL = 4;
+const NUMBER_ITEMS_1_COL = 2;
```

Comment:
We could use better names for these constants.

Reply:
fixed